support-frontend/assets/components/socialShare/socialShare.jsx

```diff
@@ -38,6 +40,16 @@ const socialMedia: {
     svg: <SvgTwitter />,
     a11yHint: 'Share on twitter',
   },
+  linkedin: {
+    link: 'https://twitter.com/intent/tweet?text=I%27ve+just+contributed+to+the+Guardian.+Join+me+in+supporting+independent+journalism+https%3A%2F%2Fsupport.theguardian.com&amp;related=guardian',
+    svg: <SvgLinkedin />,
+    a11yHint: 'Share on linkedin',
+  },
+  email: {
+    link: 'https://twitter.com/intent/tweet?text=I%27ve+just+contributed+to+the+Guardian.+Join+me+in+supporting+independent+journalism+https%3A%2F%2Fsupport.theguardian.com&amp;related=guardian',
+    svg: <SvgEmail />,
+    a11yHint: 'Share by email',
+  },
 };
 
 
```

Comment:
Not for this PR but this whole component might be a change to use the [asyncronous](https://github.com/guardian/support-frontend/blob/master/support-frontend/assets/pages/paper-subscription-checkout/components-ty/thankYou.jsx#L122) loader! it will split it out of the main js which will make the initial load time a bit smaller